plugins/rollbar/app/views/samson_rollbar/_stage_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<% if DeployGroup.enabled? %>
+  <fieldset>
+    <legend>Rollbar</legend>
+    <% stage = form.object %>
+    <% stage.rollbar_webhooks.build %>
+
+    <div class="col-lg-offset-2">
+      <%= form.fields_for :rollbar_webhooks do |rollbar_field| %>
+        <div class="form-group">
+          <div class="col-lg-4">
+            <%= rollbar_field.url_field :webhook_url, class: "form-control", placeholder: "Webhook URL" %>
```

Comment:
The other plugins that define a placeholder uses. "Webhook URL" was aiming for consistency. `README.md` explains how to get things up and running.